teeworlds7/client.go

```diff
@@ -80,56 +84,66 @@ func (client *Client) SnapFindCharacter(clientId int) (character *object7.Charac
 }
 
 func NewClient() *Client {
-	client := &Client{}
-	client.SnapshotStorage = snapshot7.NewStorage()
-	client.Game.Snap = &GameSnap{}
-	client.Game.Input = &messages7.Input{}
-	client.LocalClientId = -1
-	client.LastSend = time.Now()
-	return client
+	return &Client{
+		SnapshotStorage: snapshot7.NewStorage(),
+		Game: Game{
+			Snap:  &GameSnap{},
+			Input: &messages7.Input{},
+		},
+		LocalClientId: UnknownClientId,
+		LastSend:      time.Now(),
+	}
 }
 
-func (client *Client) sendInputIfNeeded() bool {
-	diff := time.Now().Sub(client.LastSend)
+func (client *Client) sendInputIfNeeded() (sent bool, err error) {
 	send := false
 	// at least every 10hz or on change
-	if diff.Microseconds() > 1000000 {
+	// before, this was set to 1 second
+	if time.Since(client.LastSend) > 100*time.Millisecond {
```

Comment:
As far as I see functionality wise nothing changed. I agree that the time.Since is more readable. But not sure what the comment ``// before, this was set to 1 second`` means

Reply:
it means that diff.Microseconds() > 1000000 is one second and not 100 millis.